cms/djangoapps/contentstore/proctoring.py

```diff
@@ -40,7 +40,7 @@ def register_special_exams(course_key):
         return
 
     course = modulestore().get_course(course_key)
-    if not course.enable_proctored_exams and not course.enable_timed_exams:
+    if course is not None and not course.enable_proctored_exams and not course.enable_timed_exams:
```

Comment:
This need to be changed slightly incorporating the case when `course` is actually `None`.